From 5ec9bf2d042f90f0f29b7de2bb215a8cf6ba7169 Mon Sep 17 00:00:00 2001
From: Stefan Sayer <stefan.sayer@googlemail.com>
Date: Thu, 29 Dec 2022 10:14:10 +0100
Subject: MT#55831 dsm:mod_conference:set callgroup to conference channel

dsm:mod_conference:set callgroup to conference channel
(real ticket number: MT#18839)

---
 apps/dsm/mods/mod_conference/ModConference.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/apps/dsm/mods/mod_conference/ModConference.cpp b/apps/dsm/mods/mod_conference/ModConference.cpp
index 0028a14..8b2e2ce 100644
--- a/apps/dsm/mods/mod_conference/ModConference.cpp
+++ b/apps/dsm/mods/mod_conference/ModConference.cpp
@@ -117,6 +117,9 @@ static bool ConferenceJoinChannel(DSMConfChannel** dsm_chan,
 				  DSMSession* sc_sess,
 				  const string& channel_id, 
 				  const string& mode) {
+  DBG("changing callgroup for session to %s\n", channel_id.c_str());
+  sess->changeCallgroup(channel_id);
+
   bool connect_play = false;
   bool connect_record = false;
   if (mode.empty()) {
-- 
2.6.4 (Apple Git-63)

